packages/scss/src/components/calloutDisclosure/component.scss

```diff
@@ -0,0 +1,52 @@
+@use '@lucca-front/icons/src/commons/utils/icon';
+@use '@lucca-front/scss/src/commons/utils/reset';
+@use '@lucca-front/scss/src/commons/utils/a11y';
+
+@mixin component($atRoot: 'without: rule') {
+	border-radius: var(--commons-borderRadius-L);
+	background-color: var(--palettes-50, var(--palettes-grey-50));
+	color: var(--palettes-grey-800);
+	border-color: var(--palettes-300, var(--palettes-grey-300));
+	border-style: solid;
+	border-width: var(--commons-divider-width);
+
+	a,
+	.link {
+		--commons-text-link-color: var(--palettes-grey-800);
+		--commons-text-link-hover: var(--palettes-grey-700);
+	}
+
+	@at-root ($atRoot) {
+		.calloutDisclosure-summary {
+			display: flex;
+			gap: 0.75rem;
```

Comment:
`.75rem` seems to happen more and more. Maybe we should include it in our spacings ?

Reply:
@BertrandPodevin Yep, it's planned. But to do it we have to shift multiple vars which might be painful. So we wait tokens implementation to cover it with extra values like `.375` and maybe `.125`